### ISSUE TITLE
Support installation of latest version KubeEdge and run demo on keink

### ISSUE THREAD

**Issue by fisherxu** [kubeedge/keink#8]
### What would you like to be added?

The keink now can install kubeedge v1.13, and we need to support the latest versions(include edgemesh) installation in keink. 

In addition to deploying KubeEdge, we also provide developers with demos running on keink, so that developers can quickly use keink to run kubeedge, and then develop applications on KubeEdge

**Comment by arup1221** [comment]
I am interested on the project, I would like to work on the project in LFX Fall program.

**Comment by deep-poharkar** [comment]
hey @fisherxu , 

From what I gather, the main aim is to `improve keink` so it can handle the `latest KubeEdge versions`, including `edgemesh`. This will empower developers to leverage edge computing with KubeEdge effectively. Additionally, we plan to `provide demos` on keink to help developers quickly grasp KubeEdge concepts and create edge-ready applications.

Also I feel to achieve this, we would need to `update the installation scripts, configurations, and dependencies in keink` to be compatible with the latest KubeEdge release. I assume this process might require adapting some components to accommodate the new "edgemesh" feature.

Please correct me if I’m wrong.

**Comment by vishalrajofficial** [comment]
I am Vishal Raj, a final year CSE student. I'm excited about the "KubeEdge: Support latest version in keink and run demo on keink" project. As a skilled developer familiar with Kubernetes and interested in KubeEdge, I'm eager to help deploy the latest versions, including edgemesh, in keink. Additionally, I'm excited to provide valuable demos for developers to quickly run KubeEdge on keink and facilitate application development. Looking forward to making a meaningful impact!

**Comment by sarthaksarthak9** [comment]
Hello, @fisherxu  I am interested in applying for this project under this LFX term
here what I understood
Currently, keink can only install KubeEdge v1.13. However, the latest version of KubeEdge is v1.14, and there are some new features and bug fixes in this version. Therefore, it is important to support the installation of v1.14 in keink.

Could you pls provide some material so that we can get more about the project

**Comment by PratikforCoding** [comment]
Hello @fisherxu , this project is very interesting and I want to contribute to this project under LFX fall program. I am a BS Computer Science student and  `CNCF Hooghly chapter tech team member` and I have contributed to CNCF projects and want to do some major contribution in GO development field.

I watched the architecture and it is so good and I wanna part of this development. I tried to build `kubeedge v1.13.0` using `keink` but after configuring the directories as said in the README I am facing this following problem and I also checked the `pkg/build/edgeimage/internal/kube/source.go` file and crossed the configuration but the problem remains.

`ERROR: failed to build edge-image: error finding kubeedge root: could not find kubeedge source`
